gpii/node_modules/orca/orcaSettingsHandler.js

```diff
@@ -84,7 +84,7 @@
         var userSettings = require(orcaSettingsFile);
         var currentSettingsInProfile = userSettings.profiles[userSettings.general.activeProfile[PROFILE_ID]];
 
-        var applier = fluid.makeChangeApplier(currentSettingsInProfile, {
+        var applier = fluid.makeHolderChangeApplier(currentSettingsInProfile, {
```

Comment:
Good question,

ACTUALLY the tests are passing, so let me check what's going on here.